src/main/resources/openapi/audio.yaml

```diff
@@ -29,6 +29,31 @@ paths:
             schema:
               $ref: '#/components/schemas/AudioRequestDetails'
         description: Audio Request item to add
+    delete:
+      tags:
+        - Audio
+      summary: Delete Audio Request and Processed Audio
+      description: Delete Audio Request and Processed Audio
+      operationId: deleteAudioRequest
+      parameters:
+        - in: query
+          name: audioRequestId
+          required: true
+          schema:
+            $ref: '#/components/schemas/AudioRequestId'
+      responses:
+        '200':
```

Comment:
I was expecting a `204` response code - but the ticket does have `200`. Maybe worth checking with @hemantasharma1129